app/src/main/java/ip/team13/petowner/core/network/NetworkModule.kt

```diff
@@ -0,0 +1,37 @@
+package ip.team13.petowner.core.network
```

Comment:
I think it would be better if we kept all the DI modules together (`/core/`), same for the `prefModule` for easier accessibility. Especially being a single-module app.

Reply:
Or maybe have an `injection` package in `core`? Just my 2 cents